File: CPP/Word.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    string s; cin >> s;
    int upper = 0, lower = 0;
    for (char c: s) {
        isupper(c) ? upper++ : lower++;
    }
    if (lower >= upper)
        transform(s.begin(), s.end(), s.begin(), ::tolower);
    else
        transform(s.begin(), s.end(), s.begin(), ::toupper);
    cout << s << endl;
    return 0;
}

File: Python/Word.py
#code
s = input()

count_upper=0
count_lower=0
for i in range(len(s)):
    if(s[i].isupper()):
        count_upper+=1
    else:
        count_lower+=1
if(count_upper>count_lower):
    print(s.upper())
else:
    print(s.lower())